SU-SWS/Stanford-Drupal-Profile
# DEVOPS 121: Update  entity and drupal core hotfix

# READY FOR REVIEW

# Summary
- Applying the following security releases for entity and drupal core:
https://www.drupal.org/SA-CORE-2018-001
https://www.drupal.org/sa-contrib-2018-013

# Needed By (Today or Mon.)

# Urgency
- 9

# Steps to Test

1. Pull this branch
2. Run `drush make development/make/dept.make ~/Sites/devops121-dept` to be sure site builds.
3. Wait for Travis to complete and see what tests fail (besides, AJAX wait failures).

# Affected Projects or Products
- Affects all sites on sites infrastructure

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-121
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/163

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
